bedrock-connection/src/main/java/org/cloudburstmc/protocol/bedrock/BedrockSession.java

```diff
@@ -111,10 +111,11 @@ protected void onClose() {
     protected void onPacket(BedrockPacketWrapper wrapper) {
         BedrockPacket packet = wrapper.getPacket();
         this.logInbound(packet);
+
         if (packetHandler == null) {
-            log.warn("Received packet without a packet handler for {}:{}: {}", this.getSocketAddress(), this.subClientId, packet);
+            log.debug("Received packet without a packet handler for {}:{}: {}", this.getSocketAddress(), this.subClientId, packet);
```

Comment:
Might be worth doing `log.isDebug()` here as this method may be called frequently.